neo/bin/prompt.py

```diff
@@ -55,6 +55,47 @@
 FILENAME_PROMPT_HISTORY = os.path.join(PATH_USER_DATA, '.prompt.py.history')
 
 
+class NeoHistory(FileHistory):
```

Comment:
Could you please rename it to `PromptHistory`?

Reply:
Sure